convert2rhel/systeminfo.py

```diff
@@ -403,10 +414,19 @@ def corresponds_to_rhel_eus_release(self):
         For example if we detect CentOS Linux 8.4, the corresponding RHEL 8.4 is an EUS release, but if we detect
         CentOS Linux 8.5, the corresponding RHEL 8.5 is not an EUS release.
 
-        :return: Whether or not the current system has a EUS correspondent in RHEL.
+        :return: Whether or not the current system has an EUS correspondent in RHEL.
         :rtype: bool
         """
-        return "%s.%s" % (self.version.major, self.version.minor) in EUS_MINOR_VERSIONS
+        current_version = "%s.%s" % (self.version.major, self.version.minor) in EUS_MINOR_VERSIONS
+        if tool_opts.eus and current_version:
+            self.logger.info("EUS argument detected, automatically evaluating system as EUS")
+            return current_version
+
+        if "%s.%s" % (self.version.major, self.version.minor) == "8.6":
+            print("WE ARE ATOMIC 8.6")
+            return current_version
```

Comment:
I think this can be before everything else, if we detect we are in 8.6, then just let's return True and move on.

Also, remove the print

Reply:
Alright sounds good